cfme/configure/configuration/server_settings.py

```diff
@@ -167,13 +165,12 @@ class ServerInformation(Updateable, Pretty):
 
     * ServerControlForm (Server Roles):
 
-        * websocket, ems_metrics_coordinator, cockpit_ws, smartproxy,
-        * storage_metrics_collector, database_operations, smartstate, event,
-        * storage_inventory, storage_metrics_processor, web_services, automate,
-        * rhn_mirror, database_synchronization, ems_operations, ems_metrics_collector,
-        * reporting, ems_metrics_processor, scheduler, git_owner, user_interface,
-        * embedded_ansible, storage_metrics_coordinator, ems_inventory,
-        * vmdb_storage_bridge, notifier: set True/False to change the state
+        * automate, ems_metrics_coordinator, ems_metrics_collector,
+        * ems_metrics_processor, cockpit_ws, database_operations,
+        * embedded_ansible, event, git_owner, notifier,
+        * ems_inventory, ems_operations, reporting, scheduler,
+        * smartproxy, smartstate, user_interface, web_services,
+        * websocket: set True/False to change the state
```

Comment:
Also add `internet_connectivity` and `remote_consoles` here?

Reply:
They are not in the web UI, so they're not listed here.